uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/UaaOrchestratorClaimTest.java

```
@@ -0,0 +1,301 @@
+            Thread.sleep(120000);
+            claim = getZone(claimName, ServiceName.UAA);
+            assertNotNull(claim);
+            assertEquals(claimName, claim.getMetadata().getName());
+
+            zoneName = namespace + "-" + claimName;
+            String getResponse = getZoneByName(zoneName, HttpStatus.OK);
+            getZone = OBJECT_MAPPER.readValue(getResponse, OrchestratorZoneResponse.class);
+
+            assertNotNull(getResponse);
+            assertNotNull(getZone);
+            assertEquals(zoneName, getZone.getName());
+            assertNotNull(getZone.getConnectionDetails().getDashboardUri());
+            assertNotNull(getZone.getConnectionDetails().getUri());
+            assertNotNull(getZone.getConnectionDetails().getIssuerId());
+            assertEquals(uaaRequest.getSubdomain(), getZone.getConnectionDetails().getSubdomain());
+
+            JsonNode outputSecret = orchestratorUtil.readSecret(claimName, ServiceName.UAA);
+            String outputSecretData = StringEscapeUtils.unescapeJava(outputSecret.toString());
+            assertNotNull(outputSecret);
+            assertNotNull(outputSecretData);
+
+            String uaaEksUri = null;
+            Pattern pattern = Pattern.compile(UMS_EKS_ENV_URI_REG);
+            Matcher matcher = pattern.matcher(outputSecretData);
+            if (matcher.find()) {
+                uaaEksUri = matcher.group();
+            }
+            assertTrue(outputSecretData.contains(uaaEksUri));
+
+            // if claim and outputSecret(created by Orchestrator as result of successful instance creation) both are not null
+            // that means uaa zone is successfully created in uaaz service.
+
+            UaaForProvider uaaforProvider = ((UaaClaimSpec) claim.getSpec()).getUaaForProvider();
+
+            assertEquals(uaaRequest.getSubdomain(), uaaforProvider.getSubdomain());
+            assertNotNull(outputSecret.get("connectionDetails"));
+
+        } catch (Exception e) {
```

why empty catch block, remove it if not needed or log exception.

---

Added for debugging purpose Removed IT